docs/root/operations/admin.rst

```diff
@@ -9,6 +9,25 @@ modify different aspects of the server:
 * :ref:`v1 API reference <config_admin_v1>`
 * :ref:`v2 API reference <envoy_api_msg_config.bootstrap.v2.Admin>`
 
+.. attention::
+
+  The administration interface in its current form both allows destructive operations to be
+  performed (e.g., shutting down the server) as well as potentially exposes private information
+  (e.g., stats, cluster names, cert info, etc.). It is **critical** that access to the
+  administration interface is only allowed via a secure network. This involves setting up
+  an appropriate firewall or optimally only allowing access to the administration listener via
+  localhost. This can be accomplished with a v2 configuration like the following:
+
+  .. code-block:: yaml
+
+    admin:
+      access_log_path: /tmp/admin_access.log
+      address:
+        socket_address: { address: 127.0.0.1, port_value: 9901 }
```

Comment:
Should we have all example reflect local host binding by default to provide safety to copy+paste users?

Reply:
Sure, will do a PR to update.